### PR TITLE
SALTO-2793 - SalesForce: only exclude Marketo email templates

### PR DESCRIPTION
**Pull request by tomsellek**
Instead of excluding all email templates, exclude only the problematic ones.

---

Marketo is an add-on for SalesForce that, among other things, handles promotional emails. They made the... interesting choice to create an email template for every email they send, resulting in tens of thousands of email templates. This leads to a less than stellar user experience.

---
_Release Notes_: 
Email templates are no longer excluded by default.

---
_User Notifications_: 
N/A

### CODE REVIEW

**Comment by adamro** [comment]
User Notifications wise this doesn't really effect current workspaces so we shouldn't write it

File: packages/salesforce-adapter/src/config_creator.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ export const configWithCPQ = new InstanceElement(
             metadataType: 'SiteDotCom',
           },
           {
-            metadataType: 'EmailTemplate',
```
[the remark this answers is not in the document]
Comment (adamro) [reply packages/salesforce-adapter/src/config_creator.ts:80]: and change the doc cause it's not coupled with this :\

**Comment by tomsellek** [comment]
I manually created a folder + email template and here's what the NaCl file looks like:
```
salesforce.EmailTemplate TestEmailFolder_TestEmailTemplate@d {
  available = true
  description = "Test Email Template Description"
  encodingKey = "UTF-8"
  name = "Test Email Template Name"
  style = "none"
  subject = "Test Email Template Subject"
  type = "text"
  uiType = "Aloha"
  fullName = "TestEmailFolder/TestEmailTemplate"
  content = file("salesforce/Records/EmailTemplate/TestEmailFolder/TestEmailTemplate/TestEmailTemplate.email")
}
```

**Comment by adamro** [comment]
@tomsellek
This is how an actual Marketo Email Template looks like based on our previous experience. So you are spot on :)

```
salesforce.EmailTemplate MarketoEmailTemplates_Test {
  available = true
  encodingKey = "UTF-8"
  name = "Learning.Learning Test 1"
  style = "none"
  subject = "This is a test"
  type = "custom"
  uiType = "Aloha"
  fullName = "MarketoEmailTemplates/Test"
  content = file("salesforce/Records/EmailTemplate/MarketoEmailTemplates/Test.email")
}
```

**Review by adamro — APPROVED** [review]

**Pull request merged**